client/src/app/create/create.component.scss

```diff
@@ -0,0 +1,19 @@
+.panel {
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+.panel {
+  &-body {
+    margin-bottom: 15px;
+  }
+
+  &-footer {
+
+    .list-group{
```

Comment:
Spacja między .list-group a nawiasem klamrowym. To taki kosmetyczny szczegół, nie musisz go zmieniać jak ci nie przeszkadza 👍